contracts/cw20-base/helpers.ts

```diff
@@ -67,47 +78,40 @@ const useOptions = (options: Options): Network => {
     const wallet = await Secp256k1HdWallet.deserialize(encrypted, password);
     return wallet;
   };
-  
+
   const connect = async (
     wallet: Secp256k1HdWallet,
     options: Options
   ): Promise<SigningCosmWasmClient> => {
     const [{ address }] = await wallet.getAccounts();
 
-    const client = new SigningCosmWasmClient(
-      options.httpUrl,
-      address,
-      wallet,
-      hackatomOptions.gasPrice,
-      hackatomOptions.gasLimits,
-    );
-    return client;
+    const clientOptions = { prefix: options.bech32prefix, gasPrice: options.gasPrice};
+    return await SigningCosmWasmClient.connectWithSigner(options.httpUrl, wallet, clientOptions);
   };
 
-  
+
   const hitFaucet = async (
     faucetUrl: string,
     address: string,
     ticker: string
   ): Promise<void> => {
     await axios.post(faucetUrl, { ticker, address });
   }
-  
-  const setup = async (password: string, filename?: string): Promise<SigningCosmWasmClient> => {
+
+  const setup = async (password: string, filename?: string): Promise<CW20Client> => {
     const keyfile = filename || options.defaultKeyFile;
     const wallet = await loadOrCreateWallet(hackatomOptions, keyfile, password);
     const client = await connect(wallet, hackatomOptions);
+    const account = (await wallet.getAccounts())[0].address;
 
     // ensure we have some tokens
     if (options.faucetUrl) {
-      const account = await client.getAccount();
       if (!account) {
```

Comment:
This is always false now, because `account` is always a non-empty address. I.e. you never get tokens